Assets/SolarSystem/Planets/LODs/Planet_LODs.mat

```diff
@@ -336,7 +336,7 @@ Material:
       second: {r: 1, g: 0, b: 0, a: 0}
     - first:
         name: _SunPosition
-      second: {r: -0.019818218, g: -0.09801179, b: 1.495989, a: 0}
```

Comment:
I don't have a preferred method nor am I anywhere near a git expert :). One recommendation from a teammate is to commit the original versions of those files and update your PR to include that commit as well.